api/src/controllers/team.js

```diff
@@ -59,4 +61,28 @@ async function getTeam(req, res) {
 
 }
 
-module.exports = { storeHandle, getTeam };
\ No newline at end of file
+/**
+ * Handles a PATCH request to update a team's details on the endpoint /team/:id.
+ * @param {*} req request object - team id in url, json object containing atleast one field from team model.
+ * @param {*} res response object - updated team object
+ * @returns 200: returns updated team details
+ * @returns 404: team not found
+ * @returns 400: error updating team
+ */
+async function updateTeam(req, res) {
+    const { id: _id } = req.params;
+    const updates = req.body;
+    const options = {new: true};
+
+    if (!mongoose.Types.ObjectId.isValid(_id))
+    return res.status(404).send("Error: No team with that id.");
```

Comment:
nit: `return` indentation

Reply:
also, out of scope for this PR, but I'm curious about the `getTeam` controller function. Is this fully fleshed out?
https://github.com/Researchify/Researchify/blob/main/api/src/controllers/team.js#L56-L60

Reply:
out of scope for this PR, but since thunder is looking at the teams functionality, is this controller function fleshed out? https://github.com/Researchify/Researchify/blob/main/api/src/controllers/team.js#L56-L60
The skeleton was coded by @vgoh1 as a stopgap and you guys may consider implementing this as you see fit.

Reply:
Not sure about that controller. We may look to do it in the next PR.